README.md

```diff
@@ -2,6 +2,13 @@
 
 A unified publishing application for content on GOV.UK
 
+## Nomenclature
+
+  * Content - Some text (and related fields) a user wants to publish
```

Comment:
I'm not sure if we need this term - I don't think it's a concrete enough concept that we'd use.

Reply:
I was thinking that, since I mentioned the word 'content' in these definitions, it makes sense to explain that it's not just a block of text. I added a link to content-schemas to clarify further - does that help?

Reply:
Yup cool 👍